plugin/core/views.py

```diff
@@ -373,8 +372,8 @@ def show_lsp_popup(view: sublime.View, contents: str, location: int = -1, md: bo
         flags=flags,
         location=location,
         wrapper_class=wrapper_class,
-        max_width=max_width,
-        max_height=max_height,
+        max_width=int(view.em_width() * 80.0),
```

Comment:
`int(view.em_width() * 80.0)` is 575 for me so even less than previous default.

Reply:
Yes, but this seems to be about 80 characters wide, which is a good width for reading text.

Reply:
On the other hand, we use 120 characters for this codebase for example and the popup not only shows normal text but also the code. So maybe 120 would work?

Reply:
@dmilith informed me on Discord that his font size is set to 29 and his em_width() is 14.5. So that would result in a width of 120chars * 14.5px/chars = 1740px, which is about 45% of the total width of a 4K resolution screen. Sounds good to me.